packages/api/src/services/isPoolBlacklisted.ts

```diff
@@ -0,0 +1,11 @@
+const BLACKLISTED_POOL_IDS: string[] = [
+  '421613_1_1689940800_v2',
+  '421613_1_1689940801_v2',
+];
+
+export const isPoolBlacklisted = (poolId: string): boolean => {
+  if (BLACKLISTED_POOL_IDS.includes(poolId)) {
```

Comment:
```suggestion
  return Boolean(BLACKLISTED_POOL_IDS[poolId]);
```
if you follow my advice from above